plyer/platforms/android/notification.py

```diff
@@ -40,6 +44,16 @@ def _notify(self, **kwargs):
         noti.setSmallIcon(icon)
         noti.setAutoCancel(True)
 
+        # clicking the notification will open the application
+        app_context = activity.getApplication().getApplicationContext()
+        notification_intent = INTENT(app_context, PYTHONACTIVITY)
+        notification_intent.setFlags(INTENT.FLAG_ACTIVITY_SINGLE_TOP)
+        notification_intent.setAction(INTENT.ACTION_MAIN)
+        notification_intent.addCategory(INTENT.CATEGORY_LAUNCHER)
+        pending_intent = PENDINGINTENT.getActivity(
+            app_context, 0, notification_intent, 0)
```

Comment:
A lil bit nitpicky, but the linter and pep8 don't like the bracket at the end (should be on a new line).

Reply:
Thanks for the head up, I've just moved the closing parenthesis below